config/config.go

```diff
@@ -136,6 +137,7 @@ func viperCfg() {
 	viper.SetDefault("cache.time", 10)
 	viper.SetDefault("cache.storagepath", "./")
 	viper.SetDefault("cache.sufixfile", ".cache.prestd.db")
+	viper.SetDefault("pluginpath", "./lib")
```

Comment:
Why "lib" as default path name?? What about "plugin"?

Reply:
We use the standard operating system library that keeps `.so` files in the `/usr/lib` folder